marimo/_ast/load.py

```diff
@@ -225,10 +232,5 @@ def load_app(filename: Optional[str]) -> Optional[App]:
         # Security advantages of static load are lost here, but reasonable
         # fallback for now.
         _app = _dynamic_load(filename)
-        LOGGER.warning(
-            "Static loading of notebook failed; "
-            "falling back to dynamic loading. "
-            "If you can, please report this issue to the marimo team — "
-            "https://github.com/marimo-team/marimo/issues/new?template=bug_report.yaml"
-        )
+        LOGGER.warning(FAILED_LOAD_NOTEBOOK_MESSAGE)
```

Comment:
Did this get triggered? We haven't had reports yet, and was considering removing it